workspace/server.js

```diff
@@ -1,3 +1,21 @@
 import ultra from "../server.ts";
 
-await ultra();
+await ultra({
+  middleware: [
+    async (request, next) => {
```

Comment:
We should also allow the response to be mutated. I quote like Koa's `context` signature like:

```suggestion
    async ({ request, response }, next) => {    
```